topcoffea/cfg/samples_DEPRECATED_FORMAT.cfg

```diff
@@ -1,30 +0,0 @@
-path : /home/xuan/coffea/trees/
-#options : 
-#nEvents : 1000
-#year : 2016
-xsec : xsec5TeV
-verbose : 1
-
-# Introduce your samples here following the example of the form:
-# SampleName : Filename1, Filename2, Filename2_ext
-DYJetsToLL_MLL50 : DYJetsToLL_MLL_50_TuneCP5_5020GeV_amcatnloFXFX
-DYJetsToLL_M_10to50 : DYJetsToLL_M_10to50_TuneCP5_5020GeV_amcatnloFXFX
-ZZTo2L2Nu : ZZTo2L2Nu_5p02TeV
-ZZTo4L    : ZZTo4L_5p02TeV
-WZTo3LNU  : WZTo3LNU_NNPDF30_TuneCP5_5p20TeV
-WWTo2L2Nu : WWTo2L2Nu_NNPDF31_TuneCP5_5p02TeV
-tW_noFullHad    : tbarW_5f_noFullHad_TuneCP5_5p02TeV
-tbarW_noFullHad : tbarW_5f_noFullHad_TuneCP5_5p02TeV
-TT              : TT_TuneCP5_PSweights_5p02TeV
-TTsemilep       : TT_TuneCP5_PSweights_5p02TeV
-#TT_TuneCP5up    : TT_TuneCP5up_5p02TeV
-#TT_TuneCP5down  : TT_TuneCP5down_5p02TeV
-#TT_hdampUP      : TT_hdampUP_TuneCP5_5p02TeV
-#TT_hdampDOWN    : TT_hdampDOWN_TuneCP5_5p02TeV
-#TT_mtop166p5    : TT_mtop166p5_TuneCP5_5p02TeV
-#TT_mtop178p5    : TT_mtop178p5_TuneCP5_5p02TeV
-
-# Data
-#HighEGJet  : HighEGJet_jrgonzal_data5TeV_28ago2019_HighEGJet_Run2017G_17Nov2017_v2 : 5
-#SingleMuon : SingleMuon_jrgonzal_data5TeV_30ago2019_SingleMuon_Run2017G_17Nov2017_v : 5
-#DoubleMuon : DoubleMuon_Run2017G_17Nov2017
```

Comment:
Another empty file?

Reply:
This file I also removed (I think?). But if it's not actually gone, then I agree we should remove it. I'm wondering if you could point me to where you see it in the repository?

Reply:
Same comment as above, it's gone, I was just fooled by how GitHub reported it.